src/components/wizard-upload-configfile/wizard-upload-configfile.html

```diff
@@ -0,0 +1,21 @@
+<div class="col-md-6">
+  <div class="form-group has-feedback"
+       ng-class="formFeedback('{{$ctrl.details.tunnelType}}.{{$ctrl.details.property}}')">
+    <label for="{{$ctrl.details.tunnelType}}.{{$ctrl.details.property}}" class="col-md-5 control-label">
+      {{'wizard-upload-configfile.' + $ctrl.details.property + '.label' | translate}}
+    </label>
+    <load-file-button
+            id="{{$ctrl.details.tunnelType}}.{{$ctrl.details.property}}"
+            accept="{{$ctrl.details.fileExtensions}}"
+            base64="true"
+            on-loaded="$ctrl.uploadContent(content)"
+            required="{{$ctrl.details.required}}">
+      <span>{{'wizard-upload-configfile.' + $ctrl.details.property + '.select' | translate}}</span>
+    </load-file-button>
+    <ng-show data-ng-if="$ctrl.success">
```

Comment:
can't we replace this entire block with `<i class="fa fa-check" aria-hidden="true" ng-if="$ctrl.success"></i>`?

Reply:
done